src/algebra/gcd_monoid.lean

```diff
@@ -300,6 +300,44 @@ dvd_antisymm_of_normalize_eq (normalize_gcd _ _) (normalize_gcd _ _)
   (gcd_dvd_gcd (dvd_refl _) (dvd_of_associated h))
   (gcd_dvd_gcd (dvd_refl _) (dvd_of_associated h.symm))
 
+/-- Represent a divisor of `m * n` as a product of a divisor of `m` and a divisor of `n`. -/
```

Comment:
```suggestion
/-- Represent a divisor of `m * n` as a product of a divisor of `m` and a divisor of `n`.

 Note: In general, this representation is highly non-unique. -/
```
Do we have some sort of `strong_normalization_monoid` or something, where the representation will be unique?

Reply:
No. If, say, `m = n`, and the divisor is `m | m * m`, then both `1 * m` and `m * 1` are possibilities.